composer.json

```diff
@@ -66,7 +66,10 @@
             },
             "tests": [
                 "tests/phel/"
-            ]
+            ],
+            "export-directories": ["src/phel"],
```

Comment:
I liked it more when it had it own `export` section:

```
"export": {
  "source-directories": ["src/phel"],
  "target-directory": "src/PhelGenerated",
  "namespace-prefix": "PhelGenerated"
}
```

Also the `source-directories` config is never used. Did I miss something?

Reply:
Do you mean `"export-directories"`? It's used in the CommandFactory:126:
https://github.com/phel-lang/phel-lang/pull/226/files#diff-fa00b187a2c56b8ee580791058fc96b508572e50c5da75b64d9b750e8658148eR126

Reply:
Oh, miss it.